### ISSUE TITLE
"Google Meet Easy Mute" icon extension icon not updating 

### ISSUE THREAD

**Issue by peabnuts123** [010pixel/Browser-Extensions#1]
Hi there, I understand this is kind of a strange place to be providing feedback, but I just wanted to raise with you that your Firefox extension [Google Meet Easy Mute](https://addons.mozilla.org/en-US/firefox/addon/google-meet-easy-mute) is still mostly working, but some update in something somewhere a few months ago seems to have broken the indicator icon on the extension icon. It still toggles mute in the call, but the icon always stays green. This makes it pretty hard to use, and it would great to see it fixed 🙂

**Comment by peabnuts123** [comment]
cc @010pixel

**Comment by 010pixel** [comment]
> cc @010pixel



> Hi there, I understand this is kind of a strange place to be providing feedback, but I just wanted to raise with you that your Firefox extension [Google Meet Easy Mute](https://addons.mozilla.org/en-US/firefox/addon/google-meet-easy-mute) is still mostly working, but some update in something somewhere a few months ago seems to have broken the indicator icon on the extension icon. It still toggles mute in the call, but the icon always stays green. This makes it pretty hard to use, and it would great to see it fixed 🙂

Hey Jeff (@peabnuts123), thanks for highlighting it here. Let me fix it.

**Comment by peabnuts123** [comment]
Hey @010pixel - any updates on this? I'd be happy to try work on a fix myself if you like :)

**Comment by 010pixel** [comment]
> Hey @010pixel - any updates on this? I'd be happy to try work on a fix myself if you like :)

Hi Jeff, could you please email me or pass me your email and let's continue to fix this from there. Thanks for the msg.

**Comment by peabnuts123** [comment]
@010pixel Yeah, good idea! My email is  _\<snip>_. ~~Shoot me an email and I'll delete this message.~~
